nephos/fabric/ord.py

```diff
@@ -54,11 +54,11 @@ def check_ord_tls(opts):
     Returns:
         bool: True if TLS is enabled, False if TLS is disabled.
     """
-    ord_namespace = get_namespace(opts, opts["orderers"]["msp"])
+    ord_namespace = get_namespace(opts, msp="AlphaMSP")
```

Comment:
Again, I would do this by finding the MSP where we hold the orderers, regardless of what the MSP is called, otherwise we break deployment/update backward compatibility.